forum/models/base_model.py

```diff
@@ -3,42 +3,49 @@
 """
 
 from abc import ABC, abstractmethod
+from typing import Any, Dict, Optional
 
-from bson import ObjectId
+from bson import ObjectId  # type: ignore
+from pymongo.collection import Collection  # type: ignore
+from pymongo.cursor import Cursor  # type: ignore
```

Comment:
Why are we type-ignoring this? Isn't pymongo typed? https://pymongo.readthedocs.io/en/stable/examples/type_hints.html

Reply:
```
forum/mongo.py:7: error: Cannot find implementation or library stub for module named "pymongo"  [import-not-found]
forum/mongo.py:8: error: Cannot find implementation or library stub for module named "pymongo.collection"  [import-not-found]
```
getting stub [error](https://github.com/edly-io/forum/actions/runs/10320893234/job/28572494045#:~:text=forum/mongo.py%3A7,collection%22%20%20%5Bimport%2Dnot%2Dfound%5D) for these, that's why added it.

Reply:
The problem is that requirements/quality.txt is not installed as part of CI. I think ci.in should include `-r quality.txt`.

Reply:
it resolved the issue.